src/main/java/mate/academy/bookstore/model/User.java

```diff
@@ -0,0 +1,45 @@
+package mate.academy.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "user")
```

Comment:
```suggestion
@Table(name = "users")
```
Table name should be plural according to the naming convention